src/goOutline.ts

```diff
@@ -104,7 +104,13 @@ export class GoDocumentSymbolProvider implements vscode.DocumentSymbolProvider {
 		'function': vscode.SymbolKind.Function
 	};
 
-	private convertToCodeSymbols(document: vscode.TextDocument, decls: GoOutlineDeclaration[], symbols: vscode.SymbolInformation[], containerName: string): void {
+	private convertToCodeSymbols(
+		document: vscode.TextDocument,
+		decls: GoOutlineDeclaration[],
+		symbols: vscode.SymbolInformation[],
+		containerName: string,
+		byteOffsetToDocumentOffset: (offset: number) => number): void {
```

Comment:
rename the parameter from `offset` to `byteoffset` here and every other place we are talking about converting byte offset to offset